crates/autopilot/src/util/bytes.rs

```diff
@@ -0,0 +1,30 @@
+/// A thin wrapper around a collection of bytes. Provides hex debug formatting.
+#[derive(Clone, Copy, PartialEq, Eq, Hash, Default)]
+pub struct Bytes<T>(pub T);
```

Comment:
Realistically we will probably not get rid of all those basic new types for addresses, bytes, hashes and such. Should we maybe just have another crate that only contains those basics?